BCVaccineValidator/Classes/Models/DecodedQRPayload.swift

```diff
@@ -136,4 +139,18 @@ public extension DecodedQRPayload {
         return self.vc.credentialSubject.fhirBundle.entry
             .compactMap({$0.resource}).filter({$0.resourceType.lowercased() == "Immunization".lowercased()})
     }
+    
+    func isExempt() -> Bool {
+        guard let entry = self.vc.credentialSubject.fhirBundle.entry.first(where: { $0.resource.resourceType == "Condition" }) else {
```

Comment:
Just checking the 'first' entry is not enough since there may be multiple "Condition" entries present and one of which may be of Yukon's.

Reply:
Not just one but may be more.